### PR TITLE
fixed deprications 

### PR DESCRIPTION
**Pull request by Shenung**

### CODE REVIEW

File: styles/syntax-variables.less [hunk 1]
```diff
@@ -9,7 +9,7 @@
 // @syntax-selection-color: lighten(@dark-gray, 10%);
 @syntax-selection-color: rgba(127, 127, 127, 0.6);
 // @syntax-background-color: @very-dark-gray;
-@syntax-background-color: rgba(0, 0, 0, 0.55);
+@syntax-background-color: rgba(0, 0, 0, 0.5);
```
Comment (oralordos) [inline styles/syntax-variables.less]: Why does this need to change?

File: styles/base.less [hunk 2]
```diff
@@ -1,7 +1,7 @@
 @import "syntax-variables";
 @import "system.less";
 
-atom-text-editor, :host {
+atom-text-editor, atom-text-editor {
```
Comment (oralordos) [inline styles/base.less]: One of these should be removed, you don't need to have the same thing on both sides of a comma.

File: styles/base.less [hunk 3]
```diff
@@ -52,261 +52,258 @@ atom-text-editor, :host {
   }
 }
 
-atom-text-editor .search-results .marker .region,
-:host .search-results .marker .region {
+atom-text-editor .search-results .syntax--marker .region, atom-text-editor .search-results .syntax--marker .region {
```
Comment (oralordos) [inline styles/base.less]: Same here on the comment thing.

File: styles/base.less [hunk 4]
```diff
@@ -52,261 +52,258 @@ atom-text-editor, :host {
...
   background-color: transparent;
   border: 1px solid @syntax-result-marker-color;
 }
 
-atom-text-editor .search-results .marker.current-result .region,
-:host .search-results .marker.current-result .region {
+atom-text-editor .search-results .syntax--marker.current-result .region,atom-text-editor .search-results .syntax--marker.current-result .region {
```
Comment (oralordos) [inline styles/base.less]: And here too with the comma.

**Review by oralordos — CHANGES_REQUESTED** [review]
Some duplicates in the less code that need to be addressed.

File: styles/syntax-variables.less (hunk 1 above)
Comment (Shenung) [reply styles/syntax-variables.less]: because

**Review by oralordos — APPROVED** [review]

**Pull request merged**